src/kiwixconfirmbox.h

```diff
@@ -12,12 +12,13 @@ class KiwixConfirmBox : public QDialog
     Q_OBJECT
 
 public:
-    explicit KiwixConfirmBox(QString confirmTitle, QString confirmText, QWidget *parent = nullptr);
+    explicit KiwixConfirmBox(QString confirmTitle, QString confirmText, QWidget *parent = nullptr, bool okDialog = false);
```

Comment:
Default parameter is can be a mess when there is too much of them. Please remove the default valoe of okDialog.